text/0060-tasks-api.md

```diff
@@ -568,6 +568,151 @@ New task types are also added for these operations. `indexCreation`, `indexUpdat
 }
 ```
 
+#### 9. Filtering task resources
+
+The `/tasks` and `index/{indexUid}/tasks` endpoints are filterable by type and status query parameters.
+
+##### 9.1. Query parameters definition
+
+| parameter | type | required | description                         |
+|------|------|----------|----------------------------|
+| status | string | No | Possible values are all the status of a task. By default, when `status` query parameter is not set, all task statuses are returned. |
```

Comment:
Trying to think of possible use cases for this, I got the question:

Should we support the `status` as a list of statuses? Like, what if I wanted to list all the `succeeded`, `failed` at the same time?

We could support the status as an array of strings, or just a list comma-separated.

Reply:
Hi @brunoocasali, this previous [comment](https://github.com/meilisearch/specifications/pull/116#discussion_r850466987) should answer your response. The spec needs to be updated, it could be could to write `?status=succeeded,failed` or `?status=succeeded+failed`